shared/team-building/service-tab-bar.desktop.tsx

```diff
@@ -4,76 +4,155 @@ import * as Styles from '../styles'
 import {
   serviceIdToIconFont,
   serviceIdToAccentColor,
-  serviceIdToLabel,
   serviceIdToLongLabel,
   inactiveServiceAccentColor,
 } from './shared'
 import * as Constants from '../constants/team-building'
 import {ServiceIdWithContact} from '../constants/types/team-building'
 import {Props, IconProps} from './service-tab-bar'
 
-type ExtraProps = {service: ServiceIdWithContact; isActive: boolean}
-const HoverIcon = Styles.styled<typeof Kb.Icon, ExtraProps>(Kb.Icon)(props => ({
-  '&:hover': {
```

Comment:
Is tracking hover via state instead of using CSS pseudo-selectors the preferred way of doing hover styles now?

Reply:
Before this patch there was just an icon. Now there's an icon and label. Hovering over the icon _or label_ should cause the icon to light up. It wasn't clear to me how to use `&:hover` for that.